packages/lesswrong/components/posts/PostSubmit.tsx

```diff
@@ -108,7 +106,7 @@ const PostSubmit = ({
         </div>
       }
       <div className={classes.submitButtons}>
-        {currentUser.karma >= requestFeedbackKarmaLevel && document.draft!==false && <LWTooltip
+        {requestFeedbackKarmaLevel && currentUser.karma >= requestFeedbackKarmaLevel && document.draft!==false && <LWTooltip
```

Comment:
```suggestion
        {requestFeedbackKarmaLevel !== null && currentUser.karma >= requestFeedbackKarmaLevel && document.draft!==false && <LWTooltip
```
I could realistically see someone wanting to set this to 0

Reply:
good point, makes me thing that the whole null thing is not the best design 🤔. but also seems annoying to add an additional setting